### DISCUSSION TITLE
V2.10.2以后的版本不能用ipv6的节点了

### DISCUSSION THREAD

**Discussion by yeluosln** [MetaCubeX/ClashMetaForAndroid#319] [General]
V2.10.2以后的版本不能用ipv6的节点了，包括2.10.2版本在内到最新版本2.10.4都不能使用ipv6的节点

**Comment by amorous-monk** [comment]
测试了所有的版本，发现v2.10.2及以后版本，速度大幅下降，基本上处于不可用状态，
用v2.10.1则速度良好，可用性良好。
设备：华为mate30

**Reply by ali2liu** [reply]
我的华为Mate40Pro也是，鸿蒙4.2.0系统

**Comment by 5lonely** [comment]
2.10.1之后后全局不能测速，规则大部分节点不能用，2.10.1就没问题，还有机会修复嘛

**Comment by mikko-mikust** [comment]
> V2.10.2以后的版本不能用ipv6的节点了，包括2.10.2版本在内到最新版本2.10.4都不能使用ipv6的节点

我用的一直可以但得改配置文件,配置文件的ipv6地址不能加中括号(bug),它会默认给ipv6再加一层中括号。
日志
[TCP] dial xxx (match Match/) 172.19.0.1:39230 --> xxx:443 error: [[节点ipv6地址]]:端口 connect error: address [[节点ipv6地址]]:端口: missing port in address

**Comment by 5lonely** [comment]
全局代理可以测速嘛
